### DISCUSSION TITLE
Dual Sidebar (Brave-like)

### DISCUSSION THREAD

**Discussion by frandmb** [zen-browser/desktop#904] [Ideas]
### What feature would you like to see?

Brave browser [features a sidebar](https://support.brave.com/hc/en-us/articles/5181998802061-How-do-I-use-Brave-Sidebar) that's placed at the opposite side of the vertical tabs (although it can be configured to be next to them, on the same side). It uses the same length as collapsed vertical tabs

It could hold:
- Clickable Toggles for Firefox/Zen sidebar sections -which include history/bookmarks and some extensions-, which could, in the future, include the upcoming [AI Services](https://blog.mozilla.org/en/products/firefox/firefox-news/ai-services-on-firefox/)
![image](https://github.com/user-attachments/assets/25349ee7-702b-4c7e-98f7-fce11fb15645)
- Docking space for toolbar's items, the ones that can be added through the "Customize toolbar" view. Including shortcuts for navigation, some extensions, devtools and screenshots.
- ???

**Comment by zenwav** [comment]
I'd like to second this proposal. Many extensions and web apps integrate well with a sidebar, and implementing one would significantly enhance the user experience.

For instance, I use Raindrop.io for bookmark management. Having a sidebar icon that opens the Raindrop.io interface directly in a side panel, without needing to navigate to a separate webpage, would streamline my workflow considerably.

**Comment by elieven** [comment]
Yes please! It is one of two reasons I use brave at all (dev work requiring a chromium browser being the other). Please add the brave sidebar! Much better way to have a decently long list of most commonly used shortcuts and things. More compact than essentials and can be open at all times as it uses so little space and not vertically where it is more important but on the side! It is an excellent UI pattern!
